rocketpool-daemon/node/metrics-exporter.go

```diff
@@ -71,14 +71,18 @@ func runMetricsServer(ctx context.Context, sp *services.ServiceProvider, logger
 	metricsPath := "/metrics"
 	http.Handle(metricsPath, handler)
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		w.Write([]byte(`<html>
+		_, err := w.Write([]byte(`<html>
             <head><title>Rocket Pool Metrics Exporter</title></head>
             <body>
             <h1>Rocket Pool Metrics Exporter</h1>
             <p><a href='` + metricsPath + `'>Metrics</a></p>
             </body>
             </html>`,
 		))
+
+		if err != nil {
+			logger.Warn("Error replying to http request in metrics exporter", log.Err(err))
```

Comment:
Let's add some more context here, such as where the request came from and its path (prob not the entire thing because the query string can get really big).

Reply:
the path is `/`, on L73 we specifically only handle `/` with this body